docs/source/developers/java/building.rst

```diff
@@ -78,6 +78,7 @@ We can build these manually or we can use `Archery`_ to build them using a Docke
 
 Building JNI Libraries on MacOS
 -------------------------------
+Note: If you are building on Apple silicon, be sure to use a JDK version that was compiled for that architecture. 
```

Comment:
nit:
```suggestion
Note: If you are building on Apple Silicon, be sure to use a JDK version that was compiled for that architecture. 
```
is there a canonical JDK distribution we can link to or no?

Reply:
I'm not sure its canonical, but I. will add a link to the Azul download page